Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
```

Comment:
Consider removing 'using namespace std;' and using the 'std::' prefix explicitly to avoid potential naming conflicts. [important]